README.md

```diff
@@ -1,51 +1,182 @@
-# quickstart-sidecar-aws-ecs
+# Sidecar - Terraform AWS ECS
 
-## Deploy a single container sidecar on AWS ECS
+A quick start to deploy a sidecar to AWS EC2 using Terraform!
 
-This guide explains how to deploy a basic sidecar on AWS using
-the ECS service. This example provides only the necessary configuration to
-deploy a sidecar container. For a more complete example, please see the
+This guide explains how to deploy a basic sidecar on AWS using ECS Fargate.
+This example provides only the necessary configuration to
+deploy a sidecar container. For more details, please read the
 [Cyral DIY Deployment](https://cyral.com/docs/sidecars/deployment/#custom-deployment-diy)
 documentation.
 
-By following the steps of this guide, you will deploy a sidecar container using
-a Fargate instance into an ECS cluster. You'll be able to configure the sidecar
-for specific data repositories and control the infrastructure in a way that best
-suits your needs.
+By following the steps of this guide, you will deploy a sidecar to ECS Fargate.
 
-> In case you want to deploy a sidecar using AWS EC2 instead, please see
-> the [Cyral sidecar module for AWS EC2](https://github.com/cyralinc/terraform-aws-sidecar-ec2).
+> If you prefer to deploy a sidecar to AWS EC2 instead, see
+> our [Terraform AWS EC2 Quickstart](https://github.com/cyralinc/terraform-aws-sidecar-ec2).
 
-## Usage
+---
 
-1. From your control plane (https://_tenant_.app.cyral.com) create a new sidecar and select Custom as the deployment type.
-1. Save the information for the steps below.
-1. Clone this repository and go to the `sidecar_ecs` directory.
-1. Create a sidecar_values.tfvars file in with the following content:
+## Deployment
```

Comment:
Headers dont align with others, for example the EC2 quickstart
no ##Deployment, just starting with ## Architecture and also below Usage instead of Examples

Reply:
Fixed in [9a4ba96](https://github.com/cyral-quickstart/quickstart-sidecar-aws-ecs/pull/13/commits/9a4ba9647766fe557121af74f52b7793eb812f92).